feedback/views.py

```diff
@@ -580,8 +601,8 @@ def get_context_data(self, **kwargs):
         feedbacks = (
             obj_with_attrs(obj, student=student, exercise=exercise)
             for obj in self.model.objects.all()
-                .filter(student=student, exercise=exercise)
-                .order_by('-timestamp')
+            .filter(student=student, exercise=exercise)
+            .order_by('-timestamp')
```

Comment:
Did the linter warn about the indentation here? `.filter()` and `.order_by()` are chained to `all()`, but it is harder to see that when the lines break like this.

Would these alternatives be better? I started to like the second style.

```
for obj in
self.model.objects.all()
.filter(...)
.order_by(...)
```

```
for obj in (
    self.model.objects.all()
    .filter(...)
    .order_by(...)
)
```